pkgs/applications/editors/gnome-builder/default.nix

```
@@ -129,7 +129,7 @@ stdenv.mkDerivation rec {
   '';
 
   checkPhase = ''
-    export NO_AT_BRIDGE=1
+    export NO_AT_BRIDGE=1 GTK_A11Y=none
```

`NO_AT_BRIDGE` should not be needed with GTK 4 since it no longer uses AT-SPI2-ATK.

```suggestion
    GTK_A11Y=none \
```